### ISSUE TITLE
Primary navigation: Items missing when viewing specific pages

### ISSUE THREAD

**Issue by matkuliak** [crate/crate-docs-theme#511]
Hi,

This is a known issue, but I'm not sure if you've seen it happen here specifically. Feel free to disregard if you have.

When viewing these pages from [Ecosystem Catalog](https://github.com/crate/crate-clients-tools), the primary navigation breaks:

[JDBC](https://cratedb.com/docs/jdbc/)
[PHP DBAL](https://cratedb.com/docs/dbal/)
[PHP PDO](https://cratedb.com/docs/pdo/)
[Python](https://cratedb.com/docs/python/)
[SQLAlchemy](https://cratedb.com/docs/sqlalchemy-cratedb/)
[.NET Npgsql](https://cratedb.com/docs/npgsql/)

<img width="240" alt="image" src="https://github.com/user-attachments/assets/6675f2a3-7bf8-44da-ab11-3c4e4a3f9ee1">

<img width="253" alt="image" src="https://github.com/user-attachments/assets/854bf4ce-49b4-483a-b905-74e97de4950c">

**Comment by amotl** [comment]
Yeah, I can remember this anomaly is coming from an intentional hack. Thank you for raising it. The rule is »As soon as you are in the "drivers" projects, show it like that.« It is clearly confusing, and will need to be improved on the course of all modernizations we are working on. Please keep the ticket open until it has been resolved.
